scripts/test_argument_options.sh

```diff
@@ -74,13 +74,12 @@ expect_usage_error -t gmander
 LOG_FILE="/tmp/`uuidgen`"
 printf "When given bad input, prints file not found... "
 $BITSHUFFLE --input /nonexistent/nope > "$LOG_FILE" 2>&1
-if [ "`cat $LOG_FILE`" = "Error: Input file not found" ]; then
+if grep -i 'could not open' < "$LOG_FILE" > /dev/null 2>&1 ; then
     echo "PASSED"
 else
     printf "FAILED\n\n"
 
-    printf "\t'Error: Input file not found' \
-             does not match logfile\n\n"
+    echo "Expected error message, but got: $(cat $LOG_FILE)"
```

Comment:
This appears to be a duplicate of the next line

Reply:
fixed in 017105a